andela/savage_rangers-ah
# #[166240839] Users should be able to share articles across different channels

#### Usecase
[usecase link](https://docs.google.com/document/d/17DU9wCaLMOwjY6ynhosuXFu23xrPr0ZAxVamyBcbGCE/edit)
#### What does this PR do?

this PR opens up a way of sharing the article across Authors-Haven using the following social media platform.
``Facebook`` ``Twitter`` ``Linkedin`` and ``Gmail``

#### Description of Task to be completed?

Have the following endpoint working: 
POST ``/api/articles/: slug/share/where-to-share``: this endpoint will let the user be able to share an article on social media

#### How should this be manually tested?
* Clone the repo and saved it locally on your machine.
* Open the app using any Text Editor
* Run ``npm install``   to install the dependencies.
* Set up the   ``.env`` as instructed by the   ``.env-example``
* Run  ``npm  run dev``.
* Open ``Postman`` or install ``Postman``.
* check the shared article  in the postman using the following endpoint 
 ``POST`` ``/api/articles/:slug/share/where-to-share``.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166240839](https://www.pivotaltracker.com/story/show/166240839)
